ambry-api/src/main/java/com.github.ambry/store/Transformer.java

```diff
@@ -26,6 +28,15 @@
    * @return the output {@link TransformationOutput}.
    */
   TransformationOutput transform(Message message);
+
+  /**
+   * Warms up transformer with message infos representing messages
+   * it will transform later
+   * @param messageInfos message infos that will be used to warmup transformer,
+   *                     each mesage info
```

Comment:
incomplete?

Reply:
how's it incomplete? it's an interface so it's a generalized description

Reply:
"each message info"?